src/service/NylasService.ts

```diff
@@ -137,6 +137,24 @@ export class NylasService extends APIClient {
             }
         });
     }
+
+    /**
+     * This method returns all existing Scheduler pages for the specified nylas account
+     * @param accountId
+     */
+    async getSchedulerPages(accountId: string): Promise<AxiosResponse> {
```

Comment:
Bitte nicht mit get sondern mit fetch prefixen

Reply:
Und bitte noch die richtige Response als Modal bestimmen

Reply:
Das Model hier ist etwas zu generisch. Außerdem kannst du dir das sparen, wenn du das bei invokeApi angibst (siehe unten)

```suggestion
    async getSchedulerPages(accountId: string) {
```